teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,28 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val fileRevisionBeforeChange: String?
+    val fileRevisionAfterChange: String?
+    val changeType: ChangeType
+    val filePath: String?
+    val relativeFilePath: String?
```

Comment:
What is the base directory for `relativeFilePath`, is it checkout directory? If so it makes sense to describe this in KDoc. And what is `filePath`? Is it an absolute path?